### PR TITLE
Add Chinese README.md

### PR DESCRIPTION
**Pull request by DavdGao**
## Description

Add Chinese README.md.

## Checklist

Please check the following items before code is ready to be reviewed.

- [x]  Code has passed all tests
- [x]  Docstrings have been added/updated in Google Style
- [x]  Documentation has been updated
- [x]  Code is ready for review

### CODE REVIEW

File: README_ZH.md [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+# 在mac上
+pip install -e .\[distribute\]
+```
+
+### 使用pip
+
+- 使用以下命令安装最新发布的AgentScope。
+
+```bash
+pip install AgentScope
```
Comment (pan-x-c) [inline README_ZH.md:88]: -> `pip install agentscope`

File: README_ZH.md [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+#### 内置资源
+
+AgentScope提供丰富的内置资源以便开发人员轻松构建自己的应用程序。更多内置Agent、Service和Example即将推出！
+
+##### Agent Pool
+
+- UserAgent
+- DialogAgent
+- DictDialogAgent
+- RpcDialogAgent
```
Comment (pan-x-c) [inline README_ZH.md:285]: Remove RpcDialogAgent

File: README_ZH.md [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+
+[![](https://img.shields.io/badge/license-Apache--2.0-black)](./LICENSE)
+[![](https://img.shields.io/badge/Contribute-Welcome-green)](https://modelscope.github.io/agentscope/tutorial/contribute.html)
+
+AgentScope是一款全新的Multi-Agent框架，专为应用开发者打造，旨在提供高易用、高可靠的编程体验！
+
+- **高易用**：AgentScope支持纯Python编程，提供多种语法工具实现灵活的应用流程编排，内置丰富的API服务（Service）以及应用样例，供开发者直接使用。同时，AgentScope提供了详尽的[教程](https://modelscope.github.io/agentscope/)，[API文档](https://modelscope.github.io/agentscope/)和[应用样例](https://modelscope.github.io/agentscope/)。
+
```
Comment (qbc2016) [inline README_ZH.md:16]: the above urls are the same one?

File: README_ZH.md [hunk 4]
```diff
@@ -0,0 +1,352 @@
...
+- **高鲁棒**：确保开发便捷性和编程效率的同时，针对不同能力的大模型，AgentScope提供了全面的重试机制、定制化的容错控制和面向Agent的异常处理，以确保应用的稳定、高效运行；
+
+- **基于Actor的分布式机制**：AgentScope设计了一种新的基于Actor的分布式机制，实现了复杂分布式工作流的集中式编程和自动并行优化，即用户可以使用中心化编程的方式完成分布式应用的流程编排，同时能够零代价将本地应用迁移到分布式的运行环境中。
+
+
+如果您觉得我们的工作对您有帮助，请引用[我们的论文](https://arxiv.org/abs/2402.14034)。
```
Comment (qbc2016) [inline README_ZH.md:22]: in English Readme, the url here is #references

File: README_ZH.md [hunk 5]
```diff
@@ -0,0 +1,352 @@
...
+    "config_name": "{配置名称}",       # 用于识别配置的名称
+    "model_type": "post_api",
+    "api_url": "https://xxx",         # 目标url
+    "headers": {                      # 需要的头信息
+      ...
+    },
+}
+```
+
+为了方便开发和调试，AgentScope在[Scripts](./scripts/README.md)目录下提供了丰富的脚本以快速部署模型服务。
```
Comment (ZiTao-Li) [inline README_ZH.md:158]: [Scripts] -> [scripts]

File: README_ZH.md [hunk 6]
```diff
@@ -0,0 +1,352 @@
...
+
+### 进阶使用
+
+#### **Pipeline**和**MsgHub**
+
+为了简化Agent间通信的构建，AgentScope提供了两种语法工具：**Pipeline**和**MsgHub**。
+
+- **Pipeline**：它允许用户轻松编写Agent间的通信。以Sequential Pipeline为例，以下两种代码等效，但是pipeline的实现方式更加简洁和优雅。
+
+  - **Without** pipeline的情况下，agent1、agent2和agent3顺序传递消息：
```
Comment (ZiTao-Li) [inline README_ZH.md:208]: **Without** -> **不使用**

File: README_ZH.md [hunk 7]
```diff
@@ -0,0 +1,352 @@
...
+    ```python
+    x1 = agent1(input_msg)
+    x2 = agent2(x1)
+    x3 = agent3(x2)
+    ```
+  - **With** pipeline对象的情况下：
```
Comment (ZiTao-Li) [inline README_ZH.md:214]: **With** -> **使用**

File: README_ZH.md [hunk 8]
```diff
@@ -0,0 +1,352 @@
...
+
+    ```python
+    from agentscope.pipelines import SequentialPipeline
+
+    pipe = SequentialPipeline([agent1, agent2, agent3])
+    x3 = pipe(input_msg)
+    ```
+  - **With** functional pipeline的情况下：
```
Comment (ZiTao-Li) [inline README_ZH.md:222]: **With** -> **使用**

File: README_ZH.md [hunk 9]
```diff
@@ -0,0 +1,352 @@
...
+
+    ```python
+    from agentscope.pipelines.functional import sequentialpipeline
+
+    x3 = sequentialpipeline([agent1, agent2, agent3], x=input_msg)
+    ```
+
+- **MsgHub**：为了方便地实现多人对话，AgentScope提供了Message Hub。
+
+  - **Without** `msghub`：实现多人对话：
```
Comment (ZiTao-Li) [inline README_ZH.md:232]: **Without** -> **不使用**

File: README_ZH.md [hunk 10]
```diff
@@ -0,0 +1,352 @@
...
+    x1 = agent1(x)
+    agent2.observe(x1)  # 消息x1应该广播给其他agent
+    agent3.observe(x1)
+
+    x2 = agent2(x1)
+    agent1.observe(x2)
+    agent3.observe(x2)
+    ```
+
+  - **With** `msghub`：在Message Hub中，来自参与者的消息将自动广播给所有其他参与者，因此在这种情况下，Agent的调用甚至不需要明确输入和输出消息，我们需要做的就是决定发言的顺序。此外，`msghub`还支持动态控制参与者，如下所示。
```
Comment (ZiTao-Li) [inline README_ZH.md:243]: **With** -> **使用**

**Pull request merged**